### ISSUE TITLE
Build failure with rely@3.2.0

### ISSUE THREAD

**Issue by bryphe** [facebookexperimental/reason-native#242]
I just tried upgrading a project to use rely@3.2.0, and am hitting a build issue:

```
info building @reason-native/rely@3.2.0@d41d8cd9
error: build failed with exit code: 1
  build log:
    # esy-build-package: building: @reason-native/rely@3.2.0
    # esy-build-package: pwd: /Users/bryphe/.esy/source/i/reason_native__s__rely__3.2.0__aff37be2
    # esy-build-package: running: 'dune' 'build' '-p' 'rely'
    File "src/rely/dune", line 11, characters 65-72:
    11 |     (libraries pastel.lib file-context-printer.lib unix junit re cli.lib)
                                                                          ^^^^^^^
    Error: Library "cli.lib" not found.
    Hint: try: dune external-lib-deps --missing -p rely @@default
    error: command failed: 'dune' 'build' '-p' 'rely' (exited with 1)
    esy-build-package: exiting with errors above...
    
  building @reason-native/rely@3.2.0
```

It looks like the `cli.lib` dependency added here: https://github.com/facebookexperimental/reason-native/blob/6d1af2d7bddf0306babc0957fd2879a18fa2b977/src/rely/dune#L11 isn't pulled in as a dependency in the `rely.json` file.

I looked on NPM and didn't see a package for `@reason-native/cli` - maybe it's included in another package? Otherwise, a fix could be to publish that package on NPM, and add `@reason-native/cli` as a dependency in `rely.json`. Or maybe there is some other way it's intended to be bundled that I'm missing.

The `--filter` command will be a nice enhancement, looking forward to it!

**Comment by bandersongit** [comment]
Thanks for filing, the package is included in the reason-native repo and should be bundled with the npm package itself, so I'm a little surprised that this is the case, I'll look into this further today.

**Comment by bryphe** [comment]
Thanks @bandersongit ! Trying to minimize my repro too, could be an issue with my config

**Comment by bryphe** [comment]
I just tried a clean clone from `hello-reason` (with rely pinned at 3.2.0) - this is failing for me with the same error: https://github.com/esy-ocaml/hello-reason/tree/rely-build-failure

So it seems like it's not specific to the project I was testing

**Comment by bandersongit** [comment]
Yeah it's reproducing for me locally as well

**Comment by kyldvs** [comment]
Thanks for the report @bryphe! We're looking into it and will get out a fix asap

**Comment by bandersongit** [comment]
Thanks again for reporting Bryan, I just published version 0.0.1-alpha of @reason-native/cli and version 3.2.1 of Rely which is building for me.

**Comment by bryphe** [comment]
Awesome, thanks @bandersongit and @kyldvs !
